specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/CommonDefinitions.json

```diff
@@ -642,7 +642,8 @@
           "enum": [
             "AzureFile",
             "EmptyDir",
-            "Secret"
+            "Secret",
+            "NfsAzureFile"
```

Comment:
`NfsAzureFile` looks a little strange with existing `AzureFile` one

Reply:
AzureFile is representing SMB Azure File. NfsAzureFile is representing NFS Azure file.